src/effects/VST/VSTEffect.cpp

```diff
@@ -2187,6 +2262,17 @@ void VSTEffect::callSetParameter(int index, float value)
    }
 }
 
+
+void VSTEffectWrapper::callSetParameterB(int index, float value)
```

Comment:
You wrote the functions to set chunks so that you avoided duplications:  the old functions invoke the `B` functions for common steps.

Do likewise for setting the parameter.  Make the return from this function boolean if necessary to enable that rewrite.

Reply:
I do not understand: I already wrote ::callSetParameterB. Do you mean, I should have it return a boolean and check the value where called?

The reason to write the "B" versions of callSetChunk, callSetParameter, callSetProgram was this: they are needed by other methods in the Wrapper, but the original versions also loop on mSlaves, and I guess the Wrapper should not be aware of them. The "B" versions are basically copies of the originals, with the loop on mSlaves removed.